favorite_files.py

```diff
@@ -175,7 +175,7 @@ def prompt_for_alias(self, name, group_name=None):
         files = Favs.obj.files['groups'][group_name] if group_name is not None else Favs.obj.files['files']
 
         for idx, f in enumerate(files):
-            if f['file'] == name:
+            if f['alias'] == name:
```

Comment:
`name` was being passed as os.path.basename already, f['file'] is full file path, so it wasn't finding the index. The file was added but the alias wan't changed.